### PR TITLE
Cleanup Python files of gops tools.

### PR DESCRIPTION
**Pull request by nsoranzo**
All tests pass for me.

### CODE REVIEW

File: tool_collections/gops/basecoverage/operation_filter.py [hunk 1]
```diff
@@ -1,8 +1,5 @@
 # runs after the job (and after the default post-filter)
-import os
-from galaxy import eggs
-from galaxy import jobs
-from galaxy.tools.parameters import DataToolParameter
+from galaxy.tools.parameters.basic import DataToolParameter
```
Comment (jmchilton) [inline tool_collections/gops/basecoverage/operation_filter.py]: Is this imported from basic anywhere else in tools? I feel like ``galaxy.tools.parameters`` is a better public API entry point for this. basic is an okay name - but it is not as ... obviously correct as ``galaxy.tools.parameters``. Also basic has too much junk in it - we can probably never refactor it but in a perfect world I would like to so I would prefer not to have more things depend on it.

Comment (nsoranzo) [inline tool_collections/gops/basecoverage/operation_filter.py]: No, it is imported only in gops tools. Anyway, I can revert this change if you prefer.

Comment (jmchilton) [inline tool_collections/gops/basecoverage/operation_filter.py]: Especially if it is only imported here - I think this should be reverted. That means maybe someday we can refactor and breakup basic.py :).

Comment (nsoranzo) [inline tool_collections/gops/basecoverage/operation_filter.py]: @jmchilton Done.

**Comment by jmchilton** [comment]
:+1:

**Comment by jmchilton** [comment]
Awesome - thanks for the fix and clean ups @nsoranzo.

**Pull request merged**